### ISSUE TITLE
4.0: Irrelevant Framework.

### ISSUE THREAD

**Issue by saintedlittle** [ScottMichaud/YandereSimulatorLauncher2#22]
Hello! You are using .Net Framework 4.5, which is not up to date and is not even downloadable.

**Comment by ScottMichaud** [comment]
Yes, the choice was intentional. At the time, I didn't need anything beyond .NET 4.5, and I wanted to use as old of a framework as possible so it would "just work" on as many PCs as possible (even if they're horribly behind on Windows Updates). If the user couldn't launch the app, then they would email YandereDev with a bug complaint, who has basically nothing to do with the launcher. (I've moved on to .NET 6+ for my personal projects that use C#.)

Now that Visual Studio 2022 dropped the .NET Framework 4.5 SDKs... yeah that could be a problem.

This branch (#23 ) will probably sit for a bit (I've changed jobs last year and there's some IP ownership clauses that require approval). But, yes, it's probably time to move up to .NET Framework 4.8.x if I can justify doing a major update (or someone pokes YandereDev and asks to take over development from me, of course).

**Comment by saintedlittle** [comment]
What do you mean by "or someone pokes at YandereDev and asks to take over the development from me, of course"?

**Comment by ScottMichaud** [comment]
I mean that I don't expect to update the launcher unless something serious-enough comes up. If someone wants to maintain the launcher, then they can. They just need to talk to YandereDev, publish a new version, and change the bug report email address.

**Comment by saintedlittle** [comment]
I understood you. thanks for answers
